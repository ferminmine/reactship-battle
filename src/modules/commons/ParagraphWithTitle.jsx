import React from 'react';
import withStyles from 'react-jss';

const styles = {
  instructionsContainer: {
    marginLeft: '20%',
    marginRight: '20%'
  },
  instructionsParagraph: {
    color: '#5c5c5c',
    fontWeight: 300,
    fontSize: '1.1em',
    marginTop: '1em'
  },
  paragraphTitle: {
    fontSize: '1.15em',
    fontWeight: 500,
    marginBottom: '0.3em',
    color: '#5288bb'
  }
};

const ParagraphWithTitle = ({ classes, title, paragraph }) => (
  <div className={classes.instructionsContainer}>
    <div className={classes.instructionsParagraph}>
      <div className={classes.paragraphTitle}> { title } </div>
        {paragraph}
    </div>
  </div>
);

export default withStyles(styles)(ParagraphWithTitle);
