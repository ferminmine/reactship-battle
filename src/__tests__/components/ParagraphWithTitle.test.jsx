import React from 'react';
import { mount } from 'enzyme';
import ParagraphWithTitle from '../../modules/commons/ParagraphWithTitle';
import renderer from 'react-test-renderer';

describe('ParagraphWithTitle Component Test', () => {

  it('matches the snapshot', () => {
    const tree = renderer.create(<ParagraphWithTitle />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders without crashing', () => {
    mount(<ParagraphWithTitle />);
  });
});
