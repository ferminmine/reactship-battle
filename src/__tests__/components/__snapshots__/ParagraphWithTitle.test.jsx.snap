// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ParagraphWithTitle Component Test matches the snapshot 1`] = `
<div
  className="ParagraphWithTitle-instructionsContainer-0-1-1"
>
  <div
    className="ParagraphWithTitle-instructionsParagraph-0-1-2"
  >
    <div
      className="ParagraphWithTitle-paragraphTitle-0-1-3"
    >
       
       
    </div>
  </div>
</div>
`;
